composer.json

```diff
@@ -81,7 +81,8 @@
     },
     "config": {
         "platform": {
-            "php": "8.1"
+            "php": "8.1",
+            "ext-intl": "8.1"
```

Comment:
Just checking if above line is needed. Generally less is better in these composer.json things.

Reply:
The Normalizer class is built into the php international extension.  Without the extension the functionality will fatal error.  I thought it'd be best to be explicit about the dependency than to have it work for some people (you install it automatically w/ docker) and not work for others.

Reply:
@bradymiller are you ok with keeping this in?

Reply:
hi @adunsulag , sorry for the delayed reply on this. I generally just use platform to force the build to be compatible with the lowest supported php version (in this case 8.1), so it does not use the php version that is on the server where the build is happening. No need for packages here. Can force dependencies in the require section, but don't do this since the build is generally not done on the actual server.

Reply:
@bradymiller php-intl isn't a package, its a php extension, you can't actually install it w/ composer, the intent is to warn anyone that does a composer install that their server is not setup properly as it is missing the php extension that OpenEMR relies upon, people can bypass it by doing: 

composer install --ignore-platform-reqs

We can remove the warning but if someone is missing the php extension anywhere that extension is used will fail.